### ISSUE TITLE
submission

### ISSUE THREAD

**Issue by dret** [inadarei/draft-prefer-transclude#3]
once you're satisfied with a first version, https://datatracker.ietf.org/submit/ is the place to go to submit the draft. after that it's time for marketing, alerting people about this and making sure you get feedback and buy-in. in your case, you should probably join http://httpwg.org/ and advertise your draft over there.

**Comment by dret** [comment]
@inadarei, have you started using this internally? or is this one pretty much abandoned now? just checking because over at https://github.com/dret/I-D/issues/92 i am starting work that adds to the same registry, so this could be an opportunity to push both drafts along.

**Comment by asbjornu** [comment]
Perhaps this draft is something we can adopt in the [Building Blocks for HTTP APIs IETF Working Group](https://datatracker.ietf.org/wg/httpapi/about/)?

**Comment by asbjornu** [comment]
It would perhaps be a good idea to discuss adoption of @evert's [`Prefer-Push`](https://github.com/evert/push-please) as well, since it deals with the same thing. The goal would be to unify the two drafts and agree on one set of mechanisms defined in one RFC instead of having two "competing" drafts.

**Comment by dret** [comment]
interesting idea. so what you're saying is you want a client to have the ability to say "i can deal with things, send me what you have/can"?

**Comment by asbjornu** [comment]
@dret, yes – one way ore another. I don't really have a preference on `Prefer: transclude` or `Prefer-Push`, but think that it would be beneficial to have @inadarei, @evert and others involved in the same discussion and hopefully reach an agreement that results in a single I-D.

**Comment by evert** [comment]
Sounds good to me!

**Comment by inadarei** [comment]
@asbjornu sorry, just to be clear. Are you asking if I am for including this draft in the HTTP APIs Building Blocks? The answer is resounding "yes". How can I help?

**Comment by asbjornu** [comment]
That is indeed what I am asking, @inadarei. We also need to figure out if and how we should combine your and @evert's drafts.
